### PR TITLE
New seat card and change bar components

### PR DESCRIPTION
**Pull request by seanclarkeguardian**
## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: src/lib/components/particles/change-bar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  let posleft = `${centre}px`  
+  let thisStyles = 
+  ` width: ${barwidth}px; ${positive ? `left: ${posleft}` : `left: ${negleft}`}`
+  let thisColor = ` bg-color--${party}`
+
+  styles = mergeStyles({ ...defaultStyles }, styles)
+
+
+  return <div className={styles.wrapper} style={`width: ${width}px`}>
+    <div className={styles.bar.concat(thisColor)} style={thisStyles}></div>
```
Comment (ajwl) [inline src/lib/components/particles/change-bar/index.jsx:27]: great - yes this bar bit is all looking good and working

File: src/lib/components/particles/change-bar/style.module.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.wrapper {
+	// width: 100%;
+	// background-color: grey;
+}
+
+.bar {
+	position: relative;
+	height: 1em;
```
Comment (ajwl) [inline src/lib/components/particles/change-bar/style.module.scss:8]: is there a world where we want this bar height to be customisable? - up to you

**Review by ajwl — COMMENTED** [review]
All looking good. But a question about the SeatCard component. Is this the first iteration of a component that will contain many sub-components - fleshing out the full Card? 

In that case I'd make it a Molecule, not a Particle. 

If it's remaining as basically just a formatted heading I'd call it `SmallTitleText`, or maybe `SeatCardTitle` (though for flexibility purposes, this formatting could be used for Cards that are not Seats etc etc). So preference for more generic name descriptive of the formatting  

Screenshot 2024-03-25 at 12 48 42

File: src/lib/components/particles/seat-card/index.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import defaultStyles from './style.module.scss'
+import { mergeStyles } from '$styles/helpers/mergeStyles'
+
+
+
+export function SeatCard({ title, children, styles }) {
```
Comment (njdehoog) [inline src/lib/components/particles/seat-card/index.jsx:6]: I'm not against having some kind of card component, but I'm not sure its current implementation warrants it.

File: src/lib/components/particles/change-bar/style.module.scss (hunk 2 above)
Comment (seanclarkeguardian) [reply src/lib/components/particles/change-bar/style.module.scss:8]: Yes good point. That was a working value.

File: src/lib/components/particles/change-bar/change-bar.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+export default {
+    title: 'Particles/ChangeBar',
+    component: ChangeBar
+}
+
+export const Default = {
+    args: {
+      fraction: 0.4,
+      party: "lab",
+      positive: true,
```
[the remark this answers is not in the document]
Comment (seanclarkeguardian) [reply src/lib/components/particles/change-bar/change-bar.stories.jsx:12]: I tried this and it complicates the other calculations so much I decided against; at some point something has to decide whether the bar is positive and some calculation has to reduce the change values to absolutes. It doesn't make more sense to do it here than in the repo, imo.

File: src/lib/components/particles/seat-card/index.jsx (hunk 3 above)
Comment (seanclarkeguardian) [reply src/lib/components/particles/seat-card/index.jsx:6]: ok I'm leaving the card for now. V happy to abandon it in favour of the same thing happening within the repo.

**Review by ajwl — APPROVED** [review]
great - let's merge

**Pull request merged**